docs/introduction.rst

```diff
@@ -1,10 +1,22 @@
 Introduction to Flatpak
 =======================
 
-Flatpak is a framework for distributing desktop applications on Linux. It
-has been created by developers who have a long history of working on the
+Flatpak is a framework for distributing desktop applications across various Linux
+distributions. It has been created by developers who have a long history of working on the
 Linux desktop, and is run as an independent open source project.
 
+Terminology
+-----------
+
+- Flatpak: The Flatpak package manager.
+- Flatpak application: These are the applications you install via the ``flatpak`` command or via a
+  different UI like GNOME Software or KDE Discover.
+- Runtime: Also called platforms, these are a collection of basic utilities needed for a
+  Flatpak application to work.
+- BaseApp: Similar to a runtime, but for frameworks like Electron.
+- Flatpak bundle: A specific single-file export format which contains a Flatpak app or runtime
+  (similar to the ``.deb`` or ``.rpm`` file format).
```

Comment:
Similar to ``.deb`` or ``.rpm``. Is it? Also, assumes Linux-specific knowledge.

Reply:
You double click or run `flatpak install` to install. That's very similar to `.deb` and `.rpm`.

I'll fix it.